src/format.h

```diff
@@ -82,14 +82,19 @@ extern const struct format_loader libxmp_loader_alm;
 extern const struct format_loader libxmp_loader_polly;
 extern const struct format_loader libxmp_loader_stc;
 #endif
+#endif /* LIBXMP_CORE_PLAYER */
 
+#ifndef LIBXMP_CORE_PLAYER
 #define NUM_FORMATS 50
 #define NUM_PW_FORMATS 43
+#else
+#define NUM_FORMATS 4
+#define NUM_PW_FORMATS 0
+#endif
```

Comment:
```suggestion
#elif !defined(LIBXMP_CORE_DISABLE_IT)
#define NUM_FORMATS 4
#define NUM_PW_FORMATS 0
#else
#define NUM_FORMATS 3
#define NUM_PW_FORMATS 0
#endif
```

Also, it might be a good idea to move `NUM_PW_FORMATS` into the `LIBXMP_NO_PROWIZARD` ifdef instead.

Reply:
Pushed 3ed45bede77bcdcf76f38c16ad023bb66b140227 based on your suggestiontion.

How does it look this time?